packages/dashboard/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@vaadin/dashboard",
+  "version": "24.6.0-alpha0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "vaadin-dashboard",
+  "license": "SEE LICENSE IN LICENSE",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/vaadin/web-components.git",
+    "directory": "packages/dashboard"
+  },
+  "author": "Vaadin Ltd",
+  "homepage": "https://vaadin.com/components",
+  "bugs": {
+    "url": "https://github.com/vaadin/web-components/issues"
+  },
+  "main": "vaadin-dashboard.js",
+  "module": "vaadin-dashboard.js",
+  "type": "module",
+  "files": [
+    "src",
+    "theme",
+    "vaadin-*.d.ts",
+    "vaadin-*.js",
+    "web-types.json",
+    "web-types.lit.json"
+  ],
+  "keywords": [
+    "Vaadin",
+    "dashboard",
+    "responsive",
+    "layout",
+    "web-components",
+    "web-component"
+  ],
+  "dependencies": {
+    "@open-wc/dedupe-mixin": "^1.3.0",
+    "@vaadin/component-base": "24.5.0-alpha8",
+    "@vaadin/vaadin-lumo-styles": "24.5.0-alpha8",
+    "@vaadin/vaadin-material-styles": "24.5.0-alpha8",
+    "@vaadin/vaadin-themable-mixin": "24.5.0-alpha8",
```

Comment:
I think we should update `main` to 24.6 before merging this PR, then we could update versions here to keep it consistent with the `version` of the package itself.

Reply:
This PR is targeting the feature branch `feat/dashboard`. Once we merge it to `main` sometime in the future, it shouldn't be a big effort to synchronize the package versions to whatever `main` happens to use at that moment.